### ISSUE TITLE
Report i18n issues here.

### ISSUE THREAD

**Issue by insertish** [revoltchat/translations#102]
If you spot untranslated strings, report them here.

**Comment by Cynycal2003** [comment]
Hello @insertish, can you please add a variable on this phrase http://weblate.insrt.uk/translate/revolt/web-app/bn/?checksum=91ba4af88f393301 [e.g, something like {{user}} or similar]? In some languages, especially in Bengali for a correct translation of this phrase, user's name must be added as first. Like an example - "@insert- কে উত্তর দিচ্ছেন" / "@insert- কে উত্তর দিচ্ছি" is the correct Translation in where the user's name will be added in the first unlike English.

**Comment by Cynycal2003** [comment]
And I'm sorry I forgot to mention another thing is that can you please add the Full sentence to here http://weblate.insrt.uk/translate/revolt/web-app/bn/?checksum=8004cedea89d9ab8? Like what @sayuri-gi has mentioned, in some languages, we need to write the "Revolt" word as first, Bengali have the same problem tho in where you have to write "Revolt" word as first to show the correct Translation of this phrase

**Comment by Rexogamer** [comment]
> [this](https://the-anime-god.is-inside.me/5ehw6peR.png) should be `Filipino` instead of `Pilipino`

Fixed in [36e4e9e](https://github.com/revoltchat/translations/commit/36e4e9e3ee7755fa251190461beb77c283992f5c)

**Comment by Rexogamer** [comment]
^ has been fixed

**Comment by askasys-code** [comment]
It's funny how the Revolt staff doesn't really want to fix the Italian translation that is spelling incorrect, because they want to keep their gender-neutral ideology narrative on a language where it can't be done.
Even accusing me of transphobia in the Revolt Lounge server, so that they would win the argument and kick me out.
https://weblate.insrt.uk/translate/revolt/web-app/it/?checksum=893a34ca390ecf8f&q=state%3Aapproved#comments

Expect to force a language distortion in the service of an ideology, is very unprofessional. 
Italian has two grammatical genders, masculine and feminine, but no neuter, just as, in the grammatical category of number, it distinguishes singular from plural, but does not have the dual, present in other languages. **_Biological sex and gender identity are different things from grammatical gender._** 

However, Revolt staff reject the facts. 
I am writing it here so that it will remain well imprinted on the Internet.

And this is supposed to be Open Source? At this point create a little company and privatize everything, so you can have your own internal policies and no one will complain.

**Comment by insertish** [comment]
Hi, I briefly saw this interaction yesterday but I didn't really engage in it, I don't know what you mean by gender-neutral ideology narrative, if you have an application that is meant to be used by a lot of people, presumably you should be able to address all of those people equally? This has nothing to do with 'ideology' or 'narratives', it's just how it should work.

You wouldn't an application say "she reacted to your post" for all users because not all users are female?

> Even accusing me of transphobia in the Revolt Lounge server, so that they would win the argument and kick me out.

I don't have enough context on this so I can't comment on it, I only read the actual language discussion part.

> Italian has two grammatical genders, masculine and feminine, but no neuter, just as, in the grammatical category of number, it distinguishes singular from plural, but does not have the dual, present in other languages.

Right, sure but we still need a solution to the problem.

What do you suggest?

I think you brought up using the masculine form for "Benvenuto" because it's 'neutral', but it's obviously not, the [dictionary says so at least](https://dictionary.cambridge.org/us/dictionary/italian-english/benvenuto).

There was a follow up discussion as well, and from what I can gather, the academic board or whatever did not provide any solutions either? I mean it's sort of up to the people to make a change then isn't it?

I wouldn't say a language can be set in stone either, things change all the time, there's so many dialects and region specific phrases, etc.

> And this is supposed to be Open Source? At this point create a little company and privatize everything, so you can have your own internal policies and no one will complain.

This isn't an internal policy, this is just the status-quo, we've never had sort of set in stone translation guidelines. If you want to contribute in solid guidelines, feel free to open an RFC or an MR on this repo adding more guidance to the README.

And besides, just because something is open source doesn't mean you just have full authority over a project. Other people need to be listened to as well, and so far the general vibe I am seeing is that more people are against you than with you.

**Comment by askasys-code** [comment]
> if you have an application that is meant to be used by a lot of people, presumably you should be able to address all of those people equally? 

Right, but those who use Revolt with the Italian interface are assumed to be Italian, and so there is no problem in using `Benvenuto`.
If there are personal problems, just use English, since it is grammatically predisposed to be neuter.
No Italian person is offended by the grammar of their own language. 


> There was a follow up discussion as well, and from what I can gather, the academic board or whatever did not provide any solutions either? I mean it's sort of up to the people to make a change then isn't it?

The academy has not provided any solution since it has already explained how Italian grammar works, so there is no solution to a problem that is not even there.
It is solely a personal matter of the individual.

The academy has expressed not to use that Cyrillic letter in official contexts (e.g. public documents, or public websites). 
However there is full freedom to change the language for personal purposes, in personal spaces (Example: One's own blog, one's own website).

Revolt is a portal intended for a large audience right? Like Discord after all.
So why is there no translation problem on Discord?

**Comment by janderedev** [comment]
> If there are personal problems, just use English, since it is grammatically predisposed to be neuter.

If you have such an issue with our translations then you're free to use english too, instead of making a massive drama over a rotated e :)

Also, from what I've gathered the character you're referring to as Cyrillic is not Cyrillic.

**Comment by askasys-code** [comment]
> If you have such an issue with our translations then you're free to use english too, instead of making a massive drama over a rotated e :)

Then why the insistence to keep a wrong Italian translation?
After today I will no insist about Revolt, I want just to explain facts here, for other people.

> Also, from what I've gathered the character you're referring to as Cyrillic is not Cyrillic.

https://en.wikipedia.org/wiki/Schwa_(Cyrillic)


Facts.


To conclude.

The use of Cyrillic on the Italian language, besides being problematic grammatically for the reasons explained before, it is also unreadable phonetically, this causes a problem for all Italians.
Seriously, how should I pronounce a letter that does not exist in my alphabet? I understand that there is also a certain will in wanting to "update" a language of a country, but how come this is only happening in the last 2 years? What about all the past centuries?  All the Italian poets and philosophers of the past who have contributed to improving the language?
Italian is not a new language, it is an already evolved language, and it has very ancient origins, Latin.

This experience will still be documented.

**Comment by SingleSlice** [comment]
![image](https://github.com/revoltchat/translations/assets/32547537/3448350f-0670-4cef-b472-4ac168ad5dd8)
i will just leave this here

**Comment by askasys-code** [comment]
So you are an activist. I get it. 
There is no discussion then.
You will let your narrative prevail regardless.

Here I have already expressed everything that needed to be said. The decision will be made by the majority, not by a minority.

Have a good life.

**Comment by SingleSlice** [comment]
Unfortunately, here, you are the minority, have a not so good life, hopefully it makes you change your mind.

**Comment by askasys-code** [comment]
> Unfortunately, here, you are the minority, have a not so good life, hopefully it makes you change your mind.

Here, certainly I am the minority, there is only me.
Revolt is yours, do what you want.
But the Italian language will not be changed in that stupid way in other places.
Also it leaks youthfulness in your words, you don't have enough awareness of the issue. "Old-ass academy" you said, nice, this just confirm everything of your behavior.
But if you and your friends have an activist mentality, there is nothing to do. 

As I said before, this will be documented.

**Comment by SingleSlice** [comment]
Yeah, so far i have encountered only one Italian on revolt being so closed minded as you are, we had this argument yesterday already, you got confronted by me and another Italian person, and it ended up with you being banned for your ignorance of the code of conduct.
Nothing more to say here, I hope.

**Comment by Andre601** [comment]
@askasys-code 
Could you just stop already please?

It's a language, not your entire life. If you dislike it, don't use or translate it is my opinion here.
This is such a minor and pointless discussion that nobody here really needs to have. It only spams the inbox of people who want to be informed about actual major issues.
Perhaps it's bad that Revolt tries to gender-neutral a language that is gendered, but is that REALLY that bad? No. It's a translation on a software on the internet. Not the entire dictionary of Italy or whatever.

I myself also have issues with the german translation having this formal tone that sounds completely out of place, but instead of yelling here about it and creating a big argument, I discussed in on the server calmly with the proposal to have it changed to a more personal, direct tone.

But either way, making such a big fuzz about it - especially here of all places - will not help you get it solved. It rather gets you blocked and ignored by a multitude of people.
So again: Calm down, maybe drink a cup of tea, take a walk and then come back once you calmed down enough, because you clearly are in an emotional state.

**Comment by askasys-code** [comment]
@Andre601  You are absolutely right, but I started with a simple report. And I discussed it very calmly on the Revolt Lounge server, however they treated me as a transphobic bigot and banned me, thus preventing me from providing explanations.
Instead, I thought I'd provide here all the necessary information that may be useful for a correct translation. GitHub has visibility, and other Italians may be interested in the issue.
But, if there are a couple of activists who instead of approving the changes reject them and stick with their narrative, then that's okay, it's not the end of the world.

**Comment by Andre601** [comment]
You should stop accusing people of things.

Whatever it was, I'm sure the punishment received was justified. Calling them "activists" show me, that you have a biased view and see them as evil individuals rather than just people with a different view.
Stop already.

**Comment by Rexogamer** [comment]
We want _everyone_, regardless of how they identify, to feel welcome on Revolt - explicitly gendering them, or trying to enforce masculine terms/forms as "neutral" runs against that (at least to me). The singular they in English isn't _entirely_ uncontroversial either (although it's been used for centuries at this point and the vast majority of people who take issue with it are transphobes anyway), but we use it because it's more inclusive and is a lot less clunky than writing "he/she" everywhere. Wherever possible, we try to either use gender neutral terms or avoid gendered language entirely.

Your attitude has been **consistently** hostile - you started off by accusing SingleSlice of "vandalising" the Italian translation, which isn't exactly assuming good faith, so why should we assume that of you? Comparing gender neutrality (and, more generally, the acknowledgement of trans/NB people) to "ideology" is a common tactic used by transphobes; in addition, several factors (including your Twitter bio) indicate that you are acting from a place of bigotry and/or bad faith.

I will *strongly* advise that you **drop this matter** - you're clearly not getting much support here, and over time I imagine the schwa will either follow the path of the singular they and gain wider adoption *or* fall out of use if a better alternative comes along (that isn't either "assume the person is masculine" (which is problematic for obvious reasons) or "use o/a", which is clunky and assumes the user falls into the traditional gender binary, which isn't particularly inclusive in my eyes).

**Comment by Rexogamer** [comment]
We want _everyone_, regardless of how they identify, to feel welcome on Revolt - explicitly gendering them, or trying to enforce masculine terms/forms as "neutral" runs against that (at least to me). The singular they in English isn't _entirely_ uncontroversial either (although it's been used for centuries at this point and the vast majority of people who take issue with it are transphobes anyway), but we use it because it's more inclusive and is a lot less clunky than writing "he/she" everywhere. Wherever possible, we try to either use gender neutral terms or avoid gendered language entirely.

Your attitude has been **consistently** hostile - you started off by accusing SingleSlice of "vandalising" the Italian translation, which isn't exactly assuming good faith, so why should we assume that of you? Comparing gender neutrality (and, more generally, the acknowledgement of trans/NB people) to "ideology" is a common tactic used by transphobes; in addition, your general tone and wording choices indicate that you are acting from a place of bad faith.

I will *strongly* advise that you **drop this matter** - you're clearly not getting much support here, and over time I imagine the schwa will either follow the path of the singular they and gain wider adoption *or* fall out of use if a better alternative comes along (that isn't either "assume the person is masculine" (which is problematic for obvious reasons) or "use o/a", which is clunky and assumes the user falls into the traditional gender binary, which isn't particularly inclusive in my eyes).

**Comment by NyaomiDEV** [comment]
Pitching in for just a moment to say I am an Italian and I have no issue with using Revolt in Italian; so that's literally a problem regarding this person and their like-minded people.

Also, there's a fair bit of ignorance here, because the schwa is literally present in a bunch of "dialects", like Neapolitan or Sicilian; so either this person can't pronounce it because they never really made an effort of understanding the entire reality of how Italian is spoken, also with particular respect to pronounciation (for example there's a lot of ways people pronounce vowels in the same words, but this person never even bothered to compare north to south), or, *probably* just because they're accusing all of you of being "activists" to save their transphobic buttocks.

Whatever the case may be, it's the people who make the language. You never really know if this becomes so commonplace that Accademia della Crusca is forced to adapt. That's literally how language is made.

Not only this, but Discord *is* private and they surely do want to capitalize off of everyone including transphobes, so no shit they will just take a dictionary and use whatever's on there to never have people complain to them or outright boycott them. Revolt is pretty much not Discord, and that's actually nice.
